packages/trader/src/App/Components/Form/TimePicker/__tests__/dialog.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Dialog from '../dialog';
+import moment from 'moment';
+
+describe('<Dialog />', () => {
+    const default_props = {
+        start_times: [moment('2023-11-21T08:00:00')],
+        end_times: [moment('2023-11-22T12:00:00')],
+        onChange: jest.fn(),
+        className: 'testClassName',
+        preClass: 'testPreClass',
+        selected_time: '23:45',
+    };
+
+    it('Should render minute and hours dialogs', () => {
+        render(<Dialog {...default_props} />);
+        const hourElement = screen.getByText('23');
+        const minuteElement = screen.getByText('45');
```

Comment:
```suggestion
        const hour_element = screen.getByText('23');
        const minute_element = screen.getByText('45');
```
@henry-deriv same for the rest variables you'll find